### ISSUE TITLE
Crash in Library on image resize where one dimension is larger than the source

### ISSUE THREAD

**Issue by ScottKemp** [CreunaFI/vips-image-editor#2]
Using WEBP-Express and Vips Image Editor and Elementor.

I have an image that is 250x75

Elementor is trying to make a "thumbnail" of the image that is 250x250

In the file class-image-editor-vips.php
   ```
 public function resize($max_w, $max_h, $crop = false)
    {

        if (($this->size['width'] == $max_w) && ($this->size['height'] == $max_h)) {
            return true;
        }
        try {
****************** This call will fail and return code not checked ***************
            $resized = $this->_resize($max_w, $max_h, $crop);
  			$this->image = $resized;
			
            return true;

...


    protected function _resize($max_w, $max_h, $crop = false)
    {
				
    ********************* this->size['width'] = 250, this->size['height'] = 75, max_x = 250, max_h = 250
        $dims = image_resize_dimensions($this->size['width'], $this->size['height'], $max_w, $max_h, $crop);
		
        if (!$dims) {
           ******************* FAILURE DETECTED HERE because of image dimensions that image_resize_dimensions does not understand (see it's source) returns  error **************
            return new WP_Error('error_getting_dimensions', __('Could not calculate resized image dimensions'), $this->file);
        }
        list($dst_x, $dst_y, $src_x, $src_y, $dst_w, $dst_h, $src_w, $src_h) = $dims;

        try {
            $resized = $this->image->crop($src_x, $src_y, $src_w, $src_h)->resize(max($dst_h / $src_h, $dst_w / $src_w));

            $this->update_size($dst_w, $dst_h);
            return $resized;
        } catch (Exception $exception) {
            return new WP_Error('crop_error', __('Failed to crop image'), $exception);
        }
    }


```

$this->image is now in an error state and further calls to 
    public function save($filename = null, $mime_type = null)

will crash 

Sorry, I am not a PHP programmer here.  C++ by trade.

**Comment by joppuyo** [comment]
Hey,

I have identified this issue and it will be fixed in the next release.

**Comment by joppuyo** [comment]
Fix available in the latest release [1.1.0](https://github.com/CreunaFI/vips-image-editor/releases/tag/v1.1.0).

**Comment by joppuyo** [comment]
I'm going to assume this is fixed, please do reopen if there is still an issue.
